pkg/internal/testhelpers/containers.go

```diff
@@ -106,8 +106,20 @@ func dbSetup(DBName string, superuser SuperuserStatus) (*pgxpool.Pool, error) {
 	return dbPool, nil
 }
 
+type stdoutLogConsumer struct {
+}
+
+func (s stdoutLogConsumer) Accept(l testcontainers.Log) {
+	if l.LogType == testcontainers.StderrLog {
+		fmt.Print(l.LogType, " ", string(l.Content))
```

Comment:
shouldn't this be Println (i.e. don't you need to add line breaks)

Reply:
@cevian `l.Content` always has a trailing `\n`, if we use `Println` we get double-whitespace between the lines